### PR TITLE
Serve iframe

### PR DESCRIPTION
**Pull request by evmiguel**
This PR demonstrates support for serving and paging through the test HTML files in the ARIA-AT repo.

[DEMO](https://drive.google.com/file/d/1sKZwNWbC02Sq6O5kR8jL8X7pFCBYKr6u/view?usp=sharing)

Instructions:
- Clone [ARIA-AT Test Repo](https://github.com/w3c/aria-at) to one directory level higher than your `aria-at-report` repo (more explicitly, `../`)
- Run `yarn dev`

### CODE REVIEW

**Comment by spectranaut** [comment]
When I try to initiate a test run, I get this:

no tests

File: client/webpack.config.js [hunk 1]
```diff
@@ -52,9 +52,12 @@ module.exports = {
         publicPath: '/',
         historyApiFallback: true,
         hotOnly: true,
-        proxy: {
-            '/api': 'http://localhost:5000/'
-        },
+        proxy: [
+            {
+                context: ['/tests', '/api'],
```
Comment (spectranaut) [inline client/webpack.config.js:57]: curious how this links up to the tests in the directory above!

**Comment by spectranaut** [comment]
Hey Erika, there is one thing that also needs to happen before this is merged. When you open the test file in the iframe, you actually have to specify which screen reader you want to perform the test for! To do this, you have to pass the "key" from the "at" table as a url query param. For example:

```
localhost:3000/aria-at/tests/combobox-autocomplete-both/test-18-close-combobox-interaction.html?at=nvda
localhost:3000/aria-at/tests/combobox-autocomplete-both/test-18-close-combobox-interaction.html?at=jaws
```

File: client/components/TestRun/index.jsx [hunk 2]
```diff
@@ -30,23 +53,76 @@ class TestRun extends Component {
...
+                    <h4 data-test="test-run-h4">Testing task: {testName}</h4>
+                </React.Fragment>
+            );
+            if (!this.state.runComplete) {
+                content = (
+                    <React.Fragment>
+                        <iframe
+                            src={`/${
+                                tests[this.state.currentTestIndex - 1].file
+                            }?at=${at_key}`}
```
Comment (evmiguel) [inline client/components/TestRun/index.jsx:90]: @spectranaut adding query param

**Pull request merged**